src/modelmanager.cpp

```diff
@@ -227,15 +227,26 @@ void processPipelineConfig(rapidjson::Document& configJson, const rapidjson::Val
         return;
     }
     SPDLOG_LOGGER_INFO(modelmanager_logger, "Reading pipeline: {} configuration", pipelineName);
-    auto itr2 = pipelineConfig.FindMember("nodes");
+    std::set<std::string> demultiplexerNodes;
+    std::set<std::string> gatheredDemultiplexerNodes;
+    std::optional<uint32_t> demultiplyCountEntry = std::nullopt;
+    auto demultiplyCountEntryIt = pipelineConfig.FindMember("demultiply_count");
+    if (demultiplyCountEntryIt != pipelineConfig.MemberEnd()) {
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Pipeline: {} does have demultiply at entry node", pipelineName);
+        demultiplyCountEntry = pipelineConfig["demultiply_count"].GetUint64();
+        demultiplexerNodes.insert(ENTRY_NODE_NAME);
+    } else {
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Pipeline: {} does not have demultiply at entry node", pipelineName);
+    }
 
-    std::vector<NodeInfo> info{
-        {NodeKind::ENTRY, ENTRY_NODE_NAME}};
+    std::vector<NodeInfo> info;
+    NodeInfo entryInfo{NodeKind::ENTRY, ENTRY_NODE_NAME, "", std::nullopt, {}, demultiplyCountEntry};
+    info.emplace_back(std::move(entryInfo));
```

Comment:
Is it possible to gather from entry manually?

Reply:
Need to verify but there is no good reason to do so. It was considered out of scope for DAGs design.

Reply:
Ok it seems that I relaxed the condition a bit too much in:
https://github.com/openvinotoolkit/model_server/pull/581/files/f8ed739b0f1bb93b525ccbf91460eb0591054728#diff-ddd25336c35406ea4f13e3fc1f72437f22c39357fe89feec795c954e0aabea5bL390

I should check if gathering node is EXIT node and only then allow for gathering from entry.